css-custom-functions/color-adjust-functions/style.css

```diff
@@ -0,0 +1,52 @@
+/* Borrowed from Miriam Suzanne's custom CSS functions post
+   https://www.oddbird.net/2025/04/11/custom-functions/  */
+@function --transparent(--color <color>, --alpha <number>: 0.8) returns <color> {
+  result: oklch(from var(--color) l c h / var(--alpha));
+}
+
+@function --lighter(--color <color>, --lightness-adjust <number>: 0.2) returns
+  <color> {
+  result: oklch(from var(--color) calc(l + var(--lightness-adjust)) c h);
+}
+
+@function --darker(--color <color>, --lightness-adjust <number>: 0.2) returns
+  <color> {
+  result: oklch(from var(--color) calc(l - var(--lightness-adjust)) c h);
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  height: inherit;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: repeating-linear-gradient(
+    -45deg,
+    transparent 0 20px,
+    black 20px 40px
```

Comment:
```suggestion
    lightgrey 20px 40px
```

I would suggest using a less harsh gradient. This one did hurt my eyes a little :)

<img width="504" height="408" alt="Image" src="https://github.com/user-attachments/assets/5fd238c1-88b0-474b-ae9c-b8ab0d9b035b" />

Reply:
OK, that's fair.